### PR TITLE
Adding comments and readme

### PR DESCRIPTION
**Pull request by michelchan**

### CODE REVIEW

File: api/src/bankerx/api/MorphirCodecs.scala [hunk 1]
```diff
@@ -6,6 +6,9 @@ import sttp.tapir.generic.auto.*
 import com.github.plokhotnyuk.jsoniter_scala.core.{given, *}
 import com.github.plokhotnyuk.jsoniter_scala.macros.{given, *}
 
+/**
+ * Creates Morphir specific JSON and Tapir schema encoders and decoders
```
Comment (DamianReeves) [inline api/src/bankerx/api/MorphirCodecs.scala:10]: This is actually Codecs for types that came from Morphir

File: api/src/bankerx/api/PublicEndpoints.scala [hunk 2]
```diff
@@ -13,7 +13,11 @@ import bankerx.API.*
 import bankerx.api.fdc3.*
 import bankerx.api.Codecs.{given, *}
 
+/**
+ * Defines how the path will look for the specific endpoints
+ */
 object PublicEndpoints:
+// You can access this endpoint with "www.[host]/api/bank/[bankId]/terms"
```
Comment (DamianReeves) [inline api/src/bankerx/api/PublicEndpoints.scala:20]: `www` is not necessarily true and isn't in our usecase. I'd say something like:
`(http|https)://[host][path]/api/bank/[bankId]/terms`

File: api/src/bankerx/api/PublicEndpoints.scala [hunk 3]
```diff
@@ -24,6 +28,8 @@ object PublicEndpoints:
 
   object fdc3:
     import bankerx.api.fdc3.*
+
+// You can access this endpoint with "www.[host]/api/fdc3/bank[bankId]"
```
Comment (DamianReeves) [inline api/src/bankerx/api/PublicEndpoints.scala:32]: Same as above

**Review by DamianReeves — CHANGES_REQUESTED** [review]

File: api/src/bankerx/api/MorphirCodecs.scala (hunk 1 above)
Comment (michelchan) [reply api/src/bankerx/api/MorphirCodecs.scala:10]: bad wording. will fix

**Review by DamianReeves — APPROVED** [review]
LGTM

**Pull request merged**